cfgov/jinja2/v1/_layouts/base.html

```diff
@@ -213,7 +213,7 @@
 
     {# TODO: Remove Else conditional when this is fully moved to wagtail. #}
     {# Check and include page-level JavaScript. #}
-    {% if page and page.media %}
+    {% if page %}
```

Comment:
Oh I see. I don't believe they do. Context is defined [here](https://github.com/cfpb/cfgov-refresh/blob/flapjack/cfgov/sheerlike/__init__.py#L90) and [here](https://github.com/cfpb/cfgov-refresh/blob/flapjack/cfgov/v1/__init__.py#L23) so as long as they don't include a `page` context processor or variable then we should be okay to have this conditional.